modules/xfem/src/base/XFEM.C

```diff
@@ -1312,6 +1312,24 @@ XFEM::getPhysicalVolumeFraction(const Elem * elem) const
   return phys_volfrac;
 }
 
+bool
+XFEM::isPointInsidePhysicalDomain(const Elem * elem, const Point & point) const
+{
+  std::map<unique_id_type, XFEMCutElem *>::const_iterator it;
+  it = _cut_elem_map.find(elem->unique_id());
+  if (it != _cut_elem_map.end())
+  {
+    XFEMCutElem * xfce = it->second;
+
+    if (xfce->isPointPhysical(point))
+      return true;
+  }
+  else
```

Comment:
So always return true if the element is at the end of the cut element map?

Reply:
Yes, I want it to return true if elem is not in the map.